django/contrib/contenttypes/tests/tests.py

```diff
@@ -264,3 +265,26 @@ def test_name_deprecation(self):
             "ContentType.name field doesn't exist any longer. Please remove it from your code."
         )
         self.assertTrue(ContentType.objects.filter(model='OldModel').exists())
+
+    @mock.patch('django.contrib.contenttypes.models.ContentTypeManager.get_or_create')
+    @mock.patch('django.contrib.contenttypes.models.ContentTypeManager.get')
+    def test_message_if_get_for_model_fails(self, mocked_get, mocked_get_or_create):
+        """
+        Check that `RuntimeError` with nice error message is raised if
+        `get_for_model` fails because of database errors.
+        """
+
+        def _test_message(mocked_method):
+            for ExceptionClass in (IntegrityError, OperationalError, ProgrammingError):
+                mocked_method.side_effect = ExceptionClass
+                with self.assertRaisesRegexp(
```

Comment:
assertRaisesRegexp is deprecated on Python 3, you use `self.assertRaisesMessage` I think